py/packages/genkit/src/genkit/core/types.py

```diff
@@ -1,38 +1,45 @@
 # Copyright 2025 Google LLC
 # SPDX-License-Identifier: Apache-2.0
 
-
 # generated by datamodel-codegen:
 #   filename:  genkit-schema.json
-#   timestamp: 2025-01-17T17:30:26+00:00
+#   version:   0.26.5
 
 from __future__ import annotations
 
 from enum import Enum
-from pydantic import ConfigDict, BaseModel, Field
-from typing import Any, Dict, List, Optional, Union
+from typing import Any
 
+from pydantic import BaseModel, ConfigDict, Field, RootModel
+
+
+class Model(RootModel[Any]):
```

Comment:
Could you please double check, but I believe this one was causing runtime errors before. Had to manually remove it.